docs/ref/templates/builtins.txt

```diff
@@ -1831,7 +1858,20 @@ For example, you can apply ``escape`` to fields when :ttag:`autoescape` is off:
         {{ title|escape }}
     {% endautoescape %}
 
-To escape each element of a sequence, use the :tfilter:`escapeseq` filter.
+.. note::
+
+    As mentioned in the :ttag:`autoescape` section, when filters including
+    ``escape`` are chained together, it can result in unexpected
+    outcomes if preceding filters mark a potentially unsafe string as safe due
+    to the lack of escaping caused by :ttag:`autoescape` being ``off``.
```

Comment:
Wrap at 79 chars:
```suggestion
    As mentioned in the :ttag:`autoescape` section, when filters including
    ``escape`` are chained together, it can result in unexpected outcomes if
    preceding filters mark a potentially unsafe string as safe due to the lack
    of escaping caused by :ttag:`autoescape` being ``off``.
```